tests/integration/viewer.js

```diff
@@ -249,10 +259,12 @@ describe('Viewer responds to API requests', () => {
 	/**
 	* Load another Url to the screen this scheduled for the minute after next
 	*
+	* Run this in a fresh tab it'll be fine continuing where it left off
```

Comment:
The problem was webdriver or selenium was losing access to the tab somehow. But the test would pass if it was the only test running.

This allows me to run it in a fresh tab with fresh selenium hooks which will not fail.

(Also has the benefit of testing that a tab can go down and come back and keep it's id)